content/guides/hmr-react.md

```diff
@@ -157,6 +157,21 @@ module.exports = {
 };
 ```
 
+Important to note:
+
+If you have a `query` section for the `babel-loader` in your `rules`, remember to turn off the `modules` tranformation as well, otherwise HMR won't work.
+
+```js
+loader: "babel-loader",
+query: {
```

Comment:
@Mr-Binary thanks for your contribution! 

This guide is targeted towards latest version of webpack where I believe `query` is deprecated in favor of `options`. Also, this guide already mentions this in the [babel config](https://webpack.js.org/guides/hmr-react/#babel-config) section... maybe just a short warning pointing users back to the config section would be enough? For example:

```md
W> If you are using the `options` key for your babel configuration, make sure you turn babel's `modules` feature off as mentioned in the [config section](https://webpack.js.org/guides/hmr-react/#babel-config).
```

However, I'm not sure this is even necessary with the whole config section being there. I think we usually tend to recommend using separate configs (like a `.babelrc`) instead trying to inline everything with `options`. @bebraw @simon04 thoughts?